src/figuresClasses.ts

```diff
@@ -1,11 +1,72 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
```

Comment:
[CODE KNOWLEDGE] You don't need to manually assign the constructor parameters to the class properties. TypeScript does this automatically when you declare a constructor parameter with an access modifier. Here is an example of how you can simplify your constructor:

typescript
constructor(public color: string, public a: number, public b: number, public c: number) {}
```
This comment applies to all constructors in your classes.

Reply:
how i should fix this? delete all "public"?\